Formula/pywhat.rb

```diff
@@ -16,6 +16,7 @@ class Pywhat < Formula
   end
 
   depends_on "python@3.9"
+  depends_on "rust" # for orjson
```

Comment:
We might also want a `maturin` dependency to skip having to build it? Both probably build-only deps though.

Reply:
ah right, fixed